src/index.js

```diff
@@ -12,48 +14,45 @@ export default function loader(content) {
 
   const context = options.context || this.rootContext || (this.options && this.options.context);
 
-  let url = loaderUtils.interpolateName(this, options.name, {
+  const url = loaderUtils.interpolateName(this, options.name, {
     context,
     content,
     regExp: options.regExp,
   });
 
-  let outputPath = '';
-
-  if (options.outputPath) {
-    // support functions as outputPath to generate them dynamically
-    outputPath = (
-      typeof options.outputPath === 'function' ? options.outputPath(url) : options.outputPath
-    );
-  }
+  let outputPath = (
+    typeof options.outputPath === 'function'
+      ? options.outputPath(url)
+      : path.join((options.outputPath ? options.outputPath.replace(/\/?$/, '/') : '') + url)
```

Comment:
🙃 ?

```js
console.log(path.join('dirname', 'file.extname'))
console.log(path.join('dirname/', 'file.extname'))
console.log(path.join('dirname/', '/file.extname'))

// => 'dirname/file.extname'
```
👇 
```js
path.join(outputPath || '', url)
```
?

Reply:
@michael-ciniawsky need for `outputPath` => `dir` without trailing slash

Reply:
@michael-ciniawsky oh stupid error i use `+`, instead `,`, fixed :smile:

Reply:
Why/Where ? :)

Reply:
@michael-ciniawsky 
```js
path.join((options.outputPath ? options.outputPath.replace(/\/?$/, '/') : '') + url); // `+`
```